test/__tests__/fsresolver.spec.ts

```diff
@@ -0,0 +1,81 @@
+jest.mock('fs');
+import * as path from "path";
+import { FsParser, FsResolver, SubParser, SubResolver } from "../../src";
+import { defaultContext } from "../utils";
+import { vol } from 'memfs';
+
+describe("FsResolver", function () {
+  let instance: SubResolver;
+  let contentsParser: SubParser<string>;
+
+  beforeAll(function () {
+    contentsParser = FsParser();
+    process.chdir(__dirname);
+  });
+
+  afterEach(function () {
+    vol.reset();
+  });
+
+  describe("without root prefix", function () {
+    beforeEach(function () {
+      instance = FsResolver();
+    });
+
+    it("finds absolute paths", async function () {
+      vol.fromJSON({
+        "/file.test": "correct",
+        "./file.test": "wrong",
+      });
+
+      const paf = await instance("/file.test", defaultContext());
```

Comment:
use a more descriptive name that works in english

Reply:
Actually SOMEONE introduced a collision of names in this very spec.
`path` was a variable and a module name. I just thought it was funny and who-cares-eh